packages/component-library/components/MenuList/Menu.module.scss

```diff
@@ -13,6 +14,7 @@ $side-padding: 3/4 * $ca-grid;
   border: 1px solid $kz-color-wisteria-200;
   box-shadow: $ca-box-shadow;
   line-height: 40px;
+  border-radius: $kz-border-solid-border-radius;
```

Comment:
I think it might be better to just leave it is it was with the square border. It looks a bit glitchy when hovering because the border dissapears, at least until we figure out a better way to handle this (perhaps without needing to do `overflow: hidden` which might cause clipping issues)

![image](https://user-images.githubusercontent.com/702885/78867890-3bc18a00-7a85-11ea-83d0-8f32c106d9d8.png)

Reply:
Another example I saw (DropDown uses Menu):

![image](https://user-images.githubusercontent.com/702885/78868375-2e58cf80-7a86-11ea-9999-af3c3960856e.png)

Reply:
With a quick look I've found another example that needs to be fixed. Let's do a design review next week.
 
![image](https://user-images.githubusercontent.com/2476974/78872525-c22d9a00-7a8c-11ea-80bf-bfd0504ef271.png)

Reply:
Menulist is now back to 90deg corners: https://dev.cultureamp.design/ally/border-radius/storybook-static/iframe.html?id=menulist-react--default

Another thing to note, PEX might be looking at `MenuList` when they uplift the navigation. Something to keep an eye on.

Navbar is also fixed: https://dev.cultureamp.design/ally/border-radius/storybook-static/iframe.html?id=navigationbar-react--default